README.rst

```diff
@@ -1,7 +1,7 @@
 openedx-calc
 ============
 
-A helper library for mathematical calculations, used by the `edx-platform`_.
+A helper library for mathematical calculations and symbolic mathematics, used by the `edx-platform`_.
```

Comment:
Thanks for updating the README, and in general thanks for you thoroughness in this PR 💯